api/bazel/repository_locations.bzl

```diff
@@ -33,7 +33,7 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/envoyproxy/protoc-gen-validate/archive/" + PGV_GIT_SHA + ".tar.gz"],
     ),
     com_google_googleapis = dict(
-        # TODO(dio): Consider writing a Skylark macro for importing Google API proto.
+        # TODO(dio): Consider writing a Starlark macro for importing Google API proto.
```

Comment:
can you merge master? I think this change will disappear.

Reply:
I did, but they didn't disappear. These changes were made by the check_format.py script when I ran after the first commits.

Reply:
Can you make sure you have clang10 installed? I don't know why that would make these particular changes but they don't seem relevant to this PR.

Reply:
I do have clang10. I can undo them manually if necessary.

Reply:
They are gone now.